packages/web/src/store/application/ui/withdraw-usdc/sagas.ts

```diff
@@ -203,8 +210,52 @@ function* doWithdrawUSDC({ payload }: ReturnType<typeof beginWithdrawUSDC>) {
         )
       }
     }
-    // TODO: transfer USDC to destination associated token account
+    // TODO: math.min(amount, balance)
+    let destinationTokenAccount = destinationAddress
```

Comment:
code suggestion

```
if (!libs.solanaWeb3Manager) {
  throw new Error('No libs solanaWeb3Manager defined)
}
```

and you can remove the `!` elsewhere

Reply:
lovely ty